schemas/data/ad-count.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/data/ad-count",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Ad Count",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/metricdefinition"
+  ],
+  "description": "Describes the number of ads started during the media session.",
+  "definitions": {
+    "metricdefinition": {
+      "properties": {
+        "@id": {
+          "type": "string",
+          "const": "xdm:adCount",
+          "description": "The unique identifier of this metric."
+        },
+        "schema:name": {
+          "type": "string",
+          "const": "metric: adCount",
+          "description": "Describes the number of ads started during the media session."
+        },
+        "xdm:measurement": {
+          "type": "string",
+          "description": "How to take measures of this metric.",
+          "const": "count"
```

Comment:
????

Reply:
Taken out this file as it was using the legacy metricdefinition. Using measure instead.